common-docs/identity/sign-in.md

```diff
@@ -24,7 +24,7 @@ Your signed in status is visible only to you. In the future we may add features
 
 ## Do I already have a Microsoft Account?
 
-It is possible! Do you play Minecraft, game on an Xbox, subscribe to Office 365, or use any other Microsoft products or services? If so, you probably already have a Microsoft Account.
+It is possible! Do you play Minecraft, games on an Xbox, subscribe to Office 365, or use any other Microsoft products or services? If so, you probably already have a Microsoft Account.
```

Comment:
@ganicke I was using "game" as a verb here, making a loose affinity to gamer culture. "games" doesn't sound right to me.

Reply:
I went ahead and reverted this to "game". If you like we can reword it more generally, but in this context "game" felt right to me :).

Reply:
I see. "game" or maybe "do gaming on Xbox".